style.css

```diff
@@ -98,4 +103,273 @@ li {
   cursor: pointer;
 }
 
+.vertical-space {
+  height: 50px;
+}
+
 /* hero section ends */
+
+/* works section starts */
+.works {
+  margin-top: 50px;
+  background-color: #f7f7f9;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: auto;
+  background-color: #fff;
+  height: 130vh;
+  border: 1px solid rgb(219, 216, 216);
+  border-radius: 16px;
+  padding: 16px;
+  margin: 25px;
+}
+
+.image-fluid {
+  width: 100%;
+  display: inline-block;
+  object-fit: contain;
+}
+
+.tonic {
+  color: var(--h3-color);
+  font-size: 26px;
+  font-family: var(--font-family);
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.text {
+  margin-top: 10px;
+}
+
+.multi-post {
+  color: var(--h3-color);
+  font-size: 26px;
+  font-family: var(--font-family);
+  font-weight: 700;
+  line-height: 24px;
+}
+
+.flex-items {
+  display: flex;
+  margin-top: 10px;
+}
+
+.canopy {
+  list-style: none;
+  color: #344563 !important;
+}
+
+.works_paragraph {
+  color: var(--paragraph-color);
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.list-item {
+  margin-right: 25px;
+
+  /* background-color: var(--li-color); */
+  color: #7a869a;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.btn-container-flex {
+  margin-top: 5px;
+}
+
+.btn-sm {
+  background-color: #ebebff;
+  width: 50px;
+  text-align: center;
+  height: 25px;
+  border-radius: 5px;
+  border: none;
+  color: #b7bcff;
+}
+
+.btn-md {
+  background-color: #ebebff;
+  width: 60px;
+  text-align: center;
+  height: 25px;
+  border-radius: 5px;
+  border: none;
+  color: #b7bcff;
+}
+
+.btn-lg {
+  background-color: #ebebff;
+  width: 70px;
+  text-align: center;
+  height: 25px;
+  border-radius: 5px;
+  border: none;
+  color: #b7bcff;
+}
+
+.btn-container {
+  margin-top: 10px;
+}
+
+.btn-xxl {
+  width: 120px;
+  text-align: center;
+  height: 50px;
+  border: 1px solid #6070ff;
+  background-color: #fff;
+  color: #396df2;
+  border-radius: 10px;
+  font-weight: 500;
+  font-size: 17px;
+}
```

Comment:
- [ ] Your buttons look great 👏. In addition to what you have done so far, I would suggest you to add different button interaction styles. For example, how should they appear when the user hovers on each of them, when he/she pressed one of them, etc. You use `pseudo-classes` like `:hover`, `:active`, etc. Concerning how they should look like on different interactions, there is the section in the Figma design that shows it but let me also give a screenshot of it 👇 
![image](https://user-images.githubusercontent.com/90222110/177011808-73afa4b2-4d86-4b15-9c17-bdf5d0a06a26.png)
Please use it to style all your buttons on different user interactions 😃 .

Reply:
@BertrandConxy I have added all the button interactions except the disabled interaction. after checking the Figma design I couldn't find a button which had the disabled button interaction so I only added the other three interactions
[Portfolio.webm](https://user-images.githubusercontent.com/29442846/177122430-190a0d27-b3a3-48fc-920e-65ce193831cb.webm)
.